web/api/badges/web_buffer_svg.c

```diff
@@ -866,7 +885,7 @@ int web_client_api_request_v1_badge(RRDHOST *host, struct web_client *w, char *u
     if(!st) st = rrdset_find_byname(host, chart);
     if(!st) {
         buffer_no_cacheable(w->response.data);
-        buffer_svg(w->response.data, "chart not found", NAN, "", NULL, NULL, -1, scale, 0);
+        buffer_svg(w->response.data, "chart not found", NAN, "", NULL, NULL, -1, scale, 0, -1, -1);
```

Comment:
Why are you not using the variables `fixed_width_lbl` and `fixed_width_lbl_str` here instead `-1, -1`?

Reply:
I was studying the issue that generated it, and I think I understood.

Reply:
thinking behind is if there is any error default to self scaling to be actually able to see the error (as the caller will give space expecting the good output). But maybe i can make it fixed in all cases.